lib/y2snapper_common.pm

```diff
@@ -43,7 +44,10 @@ sub y2snapper_new_snapshot {
 }
 
 sub y2snapper_untar_testfile {
-    assert_script_run "tar -xzf /home/$username/data/yast2_snapper.tgz";
+    # Due to the product change for bsc#1085266 /root is not included in
+    # snapshots anymore
+    my $args = is_sle('<15') || is_leap('<15.0') ? '' : '-C /etc';
+    assert_script_run "tar $args /etc -xzf /home/$username/data/yast2_snapper.tgz";
```

Comment:
Are your sure that `/etc` needs to be in both line? Looks weird to have `tar -C /etc /etc -xzf [...]` for some cases.

Reply:
right, that is obviously wrong here. Updated. But then I need to test this properly again.